streaming-binlog/src/main/java/org/openmrs/analytics/model/GeneralConfiguration.java

```diff
@@ -24,6 +24,8 @@
 	@Setter
 	private LinkedHashMap<String, EventConfiguration> eventConfigurations;
 	
-	// TODO add Debezium config to the model + json
+	@Getter
+	@Setter
+	private DebeziumConfiguration debeziumConfigurations;
```

Comment:
How about not having a separate JSON config file and merge the content of `debezium_config.json` into `dbz_event_to_fhir_config.json`?

Reply:
@bashir2 thatnks for the review and feedback 
The idea of merging the debezium config into `dbz_event_to_fhir_config.json` sounds good but it brings [compilation incompatibilities ](https://pastebin.com/mvgTXMYy)when your fetching event configurations for debezium connection from the GeneralConfiguration with something like this `EventConfiguration eventConfigs = generalConfiguration.getEventConfigurations();` .
Thats why i had separated them

Reply:
@bashir2 kindly requesting for your feedback